riboSeed_NAR.tex

```diff
@@ -1,476 +1,476 @@
+
+
+riboSeed relies on a closed reference genome assembly that is sufficiently closely-related to the isolate being assembled (which can be estimated using an alignment-free approach such as the KGCAK database\cite{Wang2015b}, or a kmer based method such as Kraken\cite{Wood2014}), in which rDNA regions are assembled and assumed to be in the correct genomic context.
+
+In an ideal scenario, reference selection would consist of two steps:  isolate identification (using Kraken), and then average nucleotide identity analysis to find closest complete reference. We outline protocols for reference selection in Supplementary Information, and in riboSeed documentation at \url{http://riboseed.readthedocs.io/en/latest/REFERENCE.html}.
+
+\subsection*{Usage}
+Installation (via either conda, pip, or GitHub), installs the \texttt{ribo} program. Installation using conda also installs third-party tool dependencies, such as SPAdes, and is recommended. The riboseed pipeline can be executed with a single command, \texttt{ribo run}, or under control of the user by means of distinct commands. \texttt{ribo run} performs reannotation of rDNAs in the reference genome (\texttt{scan} command), operon inference (\texttt{select} command), and \textit{de fere novo} assembly (\texttt{seed} command).  The most commonly used parameters are accessible via the \texttt{run} command. Alternatively, the full set of parameters for riboSeed can be defined within a configuration file.
+
+\subsection*{1. Preprocessing}
+
+\textbf{\texttt{scan}}\\
+\texttt{scan} uses Barrnap (\url{https://github.com/tseemann/barrnap}) to annotate rRNAs in the reference genome, and EMBOSS's seqret\cite{Rice2000a} to create GenBank, FASTA, and GFF formatted versions of the reference genome. This preprocessing step unifies the annotation vocabulary for downstream processes.\\
+\\
+\textbf{\texttt{select}}\\
+\texttt{select} infers ribosomal operon structure from the genomic location of constituent 16S, 23S and 5S sequences. Jenks Natural Breaks algorithm is used to group rRNA annotations into likely operons on the basis of genomic coordinates, using the number of 16S annotations to set the number of breaks. Output defines individual rDNA clusters and describes component elements in a plain text file. This output can be manually adjusted before assembly if clustering does not reflect the known arrangement of operons, for example based on visualization of the annotations in a genome browser.
+
+\subsection*{2. \textit{De Fere Novo} Assembly}
+
+\textbf{\texttt{seed}}\\
+\texttt{seed} implements the algorithm described in Figure S1. Short reads for the sequenced isolate are mapped to the reference genome using BWA\cite{Li2009}. Reads that map to each annotated rDNA and its flanking regions (where the flanking regions consist o
```

Comment:
is this line needed?  Or should it be listed above?  All the commands are run like this

Reply:
Do you mean line 196? If so then yes - that line should probably go back to around line 154 or so.

I'll make the change and commit.